test/rewardsTest.js

```diff
@@ -225,12 +245,15 @@ describe("Rewards", () => {
       // Alice: 10; Bob: 90
       await rewards.payReward(100)
 
+      // Make Bob ineligible for 10 units of time
       await rewards.makeIneligible(bob, 10)
 
       // Reward only to Alice
       // Alice: 20; Bob: 90; Ineligible: 90
       await rewards.payReward(100)
 
+      // Ineligibility is set for a duration. Bob was ineligible for 10 units,
```

Comment:
*Unit* of time used is a second :)

Reply:
[b3608eb](https://github.com/keep-network/sortition-pools/pull/180/commits/b3608eb30542f14b7214e70a0a3c0c2aad345e37)